docs/tutorials/scenario-simulation/.pages

```diff
@@ -1,4 +1,3 @@
 nav:
   - planning-simulation
   - rosbag-replay-simulation.md
-  - digital-twin-simulation.md
```

Comment:
If you remove this, you also have to remove the page itself.
https://github.com/WJaworskiRobotec/autoware-documentation/blob/awsim-fixes/docs/tutorials/scenario-simulation/digital-twin-simulation.md

But I feel it's good to keep this page as it is because we should write it in the future.

Reply:
I decided to remove this page and create a directory instead, because we will have much more than one instruction about digital twin simulators. I will update the explanation in the parent page

Reply:
Thanks!

Reply:
95d9238910686b22a71b4806a5eb0552fac53ce2